src/components/EngagementDataGrid/EngagementColumns.tsx

```diff
@@ -29,31 +28,68 @@ import {
   useEnumListFilterToggle,
   useFilterToggle,
 } from '../Grid';
+import { AssignmentIcon } from '../Icons';
 import { SensitivityColumn } from '../ProjectDataGrid';
 import { Link } from '../Routing';
 import { EngagementDataGridRowFragment as Engagement } from './engagementDataGridRow.graphql';
 
 export const EngagementColumns: Array<GridColDef<Engagement>> = [
   {
-    headerName: 'Name',
-    field: 'nameProjectFirst',
+    headerName: 'Project',
+    field: 'project.name',
     ...textColumn(),
     width: 200,
-    valueGetter: (_, row) =>
-      cleanJoin(' - ', [
-        row.project.name.value,
-        row.__typename === 'LanguageEngagement'
-          ? row.language.value?.name.value
-          : row.__typename === 'InternshipEngagement'
-          ? row.intern.value?.fullName
-          : null,
-      ]),
+    valueGetter: (_, row) => {
+      return row.project.name.value;
+    },
     renderCell: ({ value, row }) => (
       <Link to={`/projects/${row.project.id}`}>{value}</Link>
     ),
     hideable: false,
+  },
+  {
+    headerName: 'Language',
+    field: 'nameProjectLast',
+    ...textColumn(),
+    width: 200,
+    valueGetter: (_, row) => {
+      return row.__typename === 'LanguageEngagement'
+        ? row.language.value?.name.value
+        : row.__typename === 'InternshipEngagement'
+        ? row.intern.value?.fullName
+        : null;
+    },
+    renderCell: ({ value, row }) => {
+      return row.__typename === 'LanguageEngagement' ? (
+        <Link to={`/languages/${row.language.value!.id}`}>{value}</Link>
+      ) : row.__typename === 'InternshipEngagement' ? (
+        <Link to={`/languages/${row.intern.value!.id}`}>{value}</Link>
+      ) : null;
+    },
+    hideable: false,
     serverFilter: ({ value }) => ({ name: value }),
   },
+  {
+    headerName: 'Engagement',
```

Comment:
Idk if this is really accurate. Each row is an engagement. What do you think about just making this column blank? and that way it can be narrower too

Also I think we should make this column the first one?